src/Entities/Player.h

```diff
@@ -263,6 +263,12 @@ class cPlayer:
 
 	// tolua_end
 
+	/** Get the seed for enchanting related PRNGs, don't use this for other purposes. */
+	inline unsigned int GetEnchantmentSeed(void) { return m_EnchantmentSeed; }
+
+	/** Set the seed for enchanting related PRNGs, don't use this for other purposes. */
+	inline void SetEnchantmentSeed(unsigned int a_EnchantmentSeed) { m_EnchantmentSeed = a_EnchantmentSeed; }
```

Comment:
"OnItemEnchanted" or "PermuteEnchantmentSeed", then the function does the permutation itself, would be better than a setter.